.github/workflows/deploy.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     name: New tag
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v2
```

Comment:
Not a big issue, but this `master` was related to the version of the `actions/checkout` code, not the wp-parsely master branch.